charts/datadog/values.yaml

```diff
@@ -510,6 +510,15 @@ datadog:
     iast:
       # datadog.asm.iast.enabled -- Enable Application Security Management Interactive Application Security Testing by injecting `DD_IAST_ENABLED=true` environment variable to all pods in the cluster
       enabled: false
+  ## OTel collector related configuration
+  otelcollector:
+    enabled: false
+    ports:
+      - containerPort: 4317
+        hostPort: 4317
+      - containerPort: 4318
+        hostPort: 4318
```

Comment:
I think network policies need also to be updated. to expose the port.

and `hostPort` doesn't need to be set by default.  see how it is done for the agent container https://github.com/DataDog/helm-charts/blob/036f328b265f2d3ceb541ad3c0349552ce3d1b44/charts/datadog/templates/_container-agent.yaml#L35-L37

Reply:
Updated.